gtf2featureAnnotation.R

```diff
@@ -224,23 +237,31 @@ if ( opt$feature_type == 'transcript' &&  all(c('transcript_id', 'transcript_ver
     anno$transcript_id <- versioned_transcripts
   } 
 }
-
+save.image()
```

Comment:
can you confirm here you want the save the current workspace to a file named ".RData" ?

Reply:
Ahh well spotted, thanks, that was for debugging.